src/app/store/Product/Product.reducer.js

```diff
@@ -23,26 +24,33 @@ const initialState = {
 const ProductReducer = (state = initialState, action) => {
     switch (action.type) {
     case UPDATE_PRODUCT_DETAILS:
-        const { product, product: { attributes, variants }, filters } = action;
+        const {
+            product,
+            product: {
+                variants: initialVariants, configurable_options, attributes, type_id
+            }
+        } = action;
+
+        const brand = getBrand(attributes);
```

Comment:
No no no no no, please no specific attributes selections, we are not a custom store, we are an abstract theme for everyone.